src/main/java/org/tdl/vireo/model/repo/impl/OrganizationRepoImpl.java

```diff
@@ -136,6 +138,21 @@ public void delete(Organization organization) {
 
         organizationRepo.delete(orgId);
     }
+    
+    @Override
+	public void restoreDefaults(Organization organization) {
```

Comment:
Not sure it is going to be this simple. I set up three organizations hierarchically. 

![screen shot 2017-09-04 at 12 53 14 pm](https://user-images.githubusercontent.com/8352733/30035526-affcca3e-9170-11e7-947a-2509bee865b8.png)

I modified each child organization, thus creating new workflow steps as original on them.

I got the following errors trying to reset Test College. It did not succeed.

![screen shot 2017-09-04 at 12 52 31 pm](https://user-images.githubusercontent.com/8352733/30035541-d7d1a124-9170-11e7-8880-be52960b1538.png)

I got the following errors trying to reset Test Department. It did not succeed.

![screen shot 2017-09-04 at 12 53 04 pm](https://user-images.githubusercontent.com/8352733/30035560-014f6572-9171-11e7-89cc-6615c143e6b7.png)

The Test Degree is the only one that was able to reset its workflow, but not sure if it cleaned up the workflow step it created as its original.